cli/src/main/java/de/jplag/cli/CLI.java

```diff
@@ -78,6 +80,8 @@ public static void main(String[] args) {
                 JPlagResult result = JPlag.run(options);
                 ReportObjectFactory reportObjectFactory = new ReportObjectFactory();
                 reportObjectFactory.createAndSaveReport(result, cli.getResultFolder());
+
+                cli.printCsv(result);
```

Comment:
I feel like the code would be a bit easier to read, if the if from l. 93 was around this method and not inside it.
But if this is the way to keep it consistent in JPlag, keep it like it is